package.json

```diff
@@ -40,6 +40,7 @@
     "zod": "3.20.6"
   },
   "devDependencies": {
+    "@nuxt/content": "^2.4.3",
```

Comment:
also small nitpick: can you make sure that we have a strict version?

Reply:
sure, will do.
thanks.